### DISCUSSION TITLE
info about peculiar sysex stuff from synths

### DISCUSSION THREAD

**Discussion by markusschloesser** [christofmuc/KnobKraft-orm#76] [Q&A]
Hi, If I stumble on info which might be important for future implementations, do you want me to post it here? And where exactly?

Examples: 

1. https://cycling74.com/forums/announcing-fauxmo-a-max-based-ensoniq-fizmo-editorlibrarian
2. http://www.midiox.com/cgi-bin/yabb/YaBB.pl?board=query;action=display;num=1117445690

**Comment by christofmuc** [comment]
Let me think - the Discussions section is not as powerful, especially it does not have a closed state, you can only Delete a discussion when you think you're done.

In the issues section, it is easier - we already have a tag for "New Device", could make one issue ticket per device and collect all information in there before it finally gets implemented. 

The only downside might be that people will shy away from a software with 100 open issues, as github displays this prominently ;-)
We could have discussions in closed tickets (then they are not flagged dominantly), but then how do we distinguish stealth tickets from really done ones? Another tag? 

What is nice that that you can alt-click a tag and exclude it from the issues view, so it is easy for me to remove devices and just see the bugs/features relevant for the next release.

**Reply by markusschloesser** [reply]
wiki?
could be per synth and include a)adaptation specific stuff and b) what we talked about: how synths need to be configured to enable using them

**Reply by christofmuc** [reply]
Agree - That's probably a better black board/note book for aggregating links and other wisdom, the issues can then be reserved for those devices that are actually being implemented and tested. I'll create an entry page.

**Comment by christofmuc** [comment]
Wiki created and access permissions set - it is open to be edited by any github user.

**Comment by christofmuc** [comment]
Closing this (which is possible now) as done
